internal/monitor/controller.go

```diff
@@ -366,12 +384,12 @@ func (cm *PodMonitorType) callControllerUnpublishVolume(node *v1.Node, volumeID
 	return err
 }
 
-// podToArrayIDs returns the array IDs used by the pod)
-// TODO: multi-array
-func (cm *PodMonitorType) podToArrayIDs(pod *v1.Pod) ([]string, error) {
-	arrayIDs := make([]string, 1)
-	arrayIDs[0] = "default"
-	return arrayIDs, nil
+// podToArrayIDs returns the array IDs used by the pod, along with pvCount, and error
+func (cm *PodMonitorType) podToArrayIDs(ctx context.Context, pod *v1.Pod) ([]string, int, error) {
+	arrayIDs := make([]string, 0)
+	pvlist, _ := K8sAPI.GetPersistentVolumesInPod(ctx, pod)
```

Comment:
Does this return an error that is being discarded? Should it not be checked?

Reply:
Here we don't need, error will result in empty pvlist. We check for error where we need to, here for now not needed. When we add multi-array support, then we will need to check error and accordingly return error.